flumeserver/flumehandler/handler.go

```diff
@@ -470,6 +470,7 @@ func getERC20Holders(ctx context.Context, w http.ResponseWriter, call *rpcCall,
 type rpcTransaction struct {
   BlockHash        *common.Hash      `json:"blockHash"`
   BlockNumber      *hexutil.Big      `json:"blockNumber"`
+  Time             *hexutil.Big      `json:"time,omitempty"`
```

Comment:
In the json, let's call this "timestamp" instead of "time" to be consistent with the way it's specified in blocks.